From 4dbc4c1da2a033c497d84a1291c46f416a9cac51 Mon Sep 17 00:00:00 2001
From: David Anes <david.anes@suse.com>
Date: Thu, 4 May 2023 11:54:02 +0200
Subject: [PATCH] Avoid the removed libxml2 entity checked field

License: LGPL-2.1-or-later OR GPL-2.0-or-later OR Apache-2.0
Upstream-Status: Backport [https://github.com/dajobe/raptor/commit/4dbc4c1da2a033c497d84a1291c46f416a9cac51]
Origin: https://github.com/dajobe/raptor/commit/4dbc4c1da2a033c497d84a1291c46f416a9cac51

Since libxml2 2.11, private symbols that were never part of the public API
have been removed. Do not access xmlEntity.checked with those releases.

---
 src/raptor_libxml.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/src/raptor_libxml.c
+++ b/src/raptor_libxml.c
@@ -249 +249 @@
-#if LIBXML_VERSION >= 20627
+#if LIBXML_VERSION >= 20627 && LIBXML_VERSION < 21100
@@ -252,0 +253 @@
+     * and was later removed in version 2.11.0
